docs/src/examples/matrix-inversion-manual.jl

```diff
@@ -159,7 +159,7 @@ dx = MOI.get.(
     model,
     DiffOpt.ForwardOutVariablePrimal(),
     x,
-)  # ∂x/∂h
+)
 
 using Test                                  #src
 @test dx ≈ [0.25 ,0.75] atol=1e-4 rtol=1e-4 #src
```

Comment:
```suggestion
@test dl_dx ≈ [0.25 ,0.75] atol=1e-4 rtol=1e-4 #src
```
Fix [the test failure](https://github.com/jump-dev/DiffOpt.jl/runs/4508768666?check_suite_focus=true)